src/main/java/com/spotinst/sdkjava/model/api/ocean/kubernetes/ApiK8sVngTag.java

```diff
@@ -0,0 +1,68 @@
+package com.spotinst.sdkjava.model.api.ocean.kubernetes;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiK8sVngTag implements IPartialUpdateEntity {
```

Comment:
parameter names should be tagKey and tagValue. Is that correct?

Reply:
Rnamed to match the name